includes/wikia/TemplateClassification.class.php

```diff
@@ -30,6 +38,15 @@ class TemplateClassification {
 		self::TEMPLATE_QUOTE,
 		self::TEMPLATE_NAVBOX,
 		self::TEMPLATE_UNCLASSIFIED,
```

Comment:
Per discussion: we need to keep UNCLASSIFIED for now because it has a slightly different meaning. Once we hook up the Template Conversion flow to the service UNCLASSIFIED will be removed.